valkyrie.gemspec

```diff
@@ -47,6 +47,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "coveralls"
   spec.add_development_dependency "bixby"
   spec.add_development_dependency 'yard'
-  spec.add_development_dependency 'solr_wrapper'
```

Comment:
I wonder if we should keep both versions. I know some people at least have trouble getting Docker to work.